src/app/[identifier]/page.tsx

```diff
@@ -85,8 +85,8 @@ export default function Page({ params }: ResponseProps) {
   return (
     <>
       <Modal isOpen={isModalOpen} toggleModal={closeModal} response={modalData} title={modalTitle} />
-      <div className="flex flex-col w-full items-center justify-center">
-        <div className="w-full flex md:flex-row flex-col justify-center items-center md:space-x-0 ">
+      <div className="flex flex-col w-full items-center justify-center mt-[25%] md:mt-[12.5%] lg:mt-[15%]">
+        <div className="w-full flex md:flex-row gap-4 flex-col justify-center items-center md:space-x-0 ">
```

Comment:
is the gap for mobile responsiveness? would prefer to remove this (nit though, just want it to be as similar to original designs)